cli/cli/scripts/deploy-ray-and-syntho-stack.sh

```diff
@@ -424,6 +427,30 @@ get_all_logs() {
     write_and_exit "$errors" "get_all_logs"
 }
 
+post_deployment() {
+    local errors=""
+    SYNTHO_CLI_PROCESS_LOGS="$SYNTHO_CLI_PROCESS_DIR/post_deployment.log"
+
+    echo "post_deployment:annotate_ingress has been started" >> $SYNTHO_CLI_PROCESS_LOGS
+    if ! annotate_ingress >> $SYNTHO_CLI_PROCESS_LOGS 2>&1; then
+        errors+="Annoting ingress has been unexpectedly failed.\n"
+    fi
+    echo "post_deployment:annotate_ingress has been finalized" >> $SYNTHO_CLI_PROCESS_LOGS
+
+    write_and_exit "$errors" "post_deployment"
+}
```

Comment:
Just curious: why the setup of annotating the deployment afterwards? We can also just include the annotation in the values for the Helm chart right?

Reply:
yes we could do it too. I think I just picked the quickest way. Would you prefer that over this?

Reply:
I noticed [this](https://github.com/syntho-ai/deployment-tools/blob/a0d955c7eea2c6ffa1ccdf96229c7326e1c0be09/helm/syntho-ui/templates/frontend/frontend-ingress.yaml#L24-L26)  and [this](https://github.com/syntho-ai/deployment-tools/blob/a0d955c7eea2c6ffa1ccdf96229c7326e1c0be09/helm/syntho-ui/values.yaml#L82-L91) and IMO trying to add that annonation selectively to directly ingress would require a bit refactoring there.

Reply:
mm I think we could add it right after [this](https://github.com/syntho-ai/deployment-tools/blob/a0d955c7eea2c6ffa1ccdf96229c7326e1c0be09/helm/syntho-ui/templates/frontend/frontend-ingress.yaml#L25)  line with a if condition 🤔

Reply:
It's fine to keep it for now as is, in the future we might want to adjust this indeed